linux_docker_resources/Dockerfile-CentOS

```diff
@@ -31,7 +30,9 @@ RUN yum install \
   python36-devel \
   python36-docutils \
   python36-pip \
+  python36-pycodestyle \
   python36-pytest-repeat \
+  python36-snowballstemmer \
```

Comment:
I don't recognize snowballstemmer. What requires this?

Reply:
It is in the dependency chain for pydocstyle, which itself isn't available in packaged form. The goal is to get as few packages as possible from pip to get a better idea of what packaging work remains.